src/cothread/cothread.py

```diff
@@ -73,6 +73,7 @@
 import traceback
 import collections
 import threading
+import thread
```

Comment:
This should just be `_thread` - as there is no module just called `thread`. I leave it up to you whether you do an `import _thread as thread` or not.

Reply:
Yeah, I messed up the original commit.  Have fixed it and force pushed the change.